src/ts/storageutils.ts

```diff
@@ -50,11 +33,14 @@ export namespace StorageUtils {
    * @return {string | null} Returns the string if found, null if there is no data stored for the key
    */
   export function getItem(key: string): string | null {
-    if (StorageUtils.hasLocalStorage()) {
-      return window.localStorage.getItem(key);
-    } else {
-      return null;
+    try {
+      if (StorageUtils.isLocalStorageAvailable()) {
+        return window.localStorage.getItem(key);
+      }
+    } catch (e) {
+      console.warn(`Failed to retrieve storage entry for ${key}`);
```

Comment:
IMO `warn` is too high of a log level, as we will always check for certain localStorage keys. I'd suggest `debug` or removing the log completely.

Reply:
Removed the log entirely as well in [Removed superfluous try-catch](https://github.com/bitmovin/bitmovin-player-ui/pull/612/commits/3c2df0690034ff24b315e7a3820e13e4970f0571). Just not a fan of empty `catch` blocks :)